conda_smithy/configure_feedstock.py

```diff
@@ -696,7 +778,25 @@ def _santize_remote_ci_setup(remote_ci_setup):
     return remote_ci_setup
 
 
-def finalize_config(config, platform, arch, forge_config):
+def finalize_config(
+    config: Dict[
+        str,
+        Union[
+            List[str],
+            OrderedDict,
+            List[Union[str, List[str]]],
+            List[List[str]],
+        ],
+    ],
```

Comment:
c.f. earlier comment to simplify this type.

Reply:
simplified it to generic `Union[list, dict]`